### ISSUE TITLE
Partial error message placeholder appears after clicking "Sign In" with no credentials

### ISSUE THREAD

**Issue by therealnicksaunders** [okta/okta-signin-widget#390]
This screenshot should be fairly self-explanatory:

![image](https://user-images.githubusercontent.com/2517229/35869372-1e099672-0b1c-11e8-83b1-fbf9df02178e.png)

**Comment by santhoshbalakrishnan** [comment]
@therealnicksaunders It does seem like the CSS has been customized for the widget. Have you tried removing your CSS changes and testing?

It seems to work fine on a fresh clone of the repo:

<img width="467" alt="screen shot 2018-02-06 at 9 04 23 am" src="https://user-images.githubusercontent.com/24683447/35872835-caf6ff5a-0b1c-11e8-8e34-324dba2551fa.png">

**Comment by inergy** [comment]
We are using the widget without any customized css for the elements in question, and Inspecting the elements does not show any content:

![image](https://user-images.githubusercontent.com/8495416/36126753-c241560c-100e-11e8-8be2-bf5ee3a332e2.png)

**Comment by hor-kanchan-okta** [comment]
@inergy Just a guess. Check for the css box-sizing. It may be overridden by other css.

**Comment by inergy** [comment]
@hor-kanchan-okta While box-sizing would resolve the squished look when the content is empty, it is not related to issue of empty text content in the p tag (shown in my first screenshot). Other error messages appear correctly (such as "Sign in Failed!" when using invalid credentials), but this specific error message is not populating the element with text.

**Comment by hor-kanchan-okta** [comment]
@inergy Which version of okta-signin-widget are you using? I recall there is a bug #313 which didn't lookup the string correctly from "courage" bundle.
"oform.errorbanner.title" is from "courage" bundle. [ErrorBanner.js#L9](https://github.com/okta/okta-signin-widget/blob/2.6/packages/@okta/courage/src/views/forms/helpers/ErrorBanner.js#L9)

**Comment by santhoshbalakrishnan** [comment]
@inergy I am having some trouble repro-ing the issue here. I've tried my best in reproducing it with the bare minimum markup and not able to hit the issue you are hitting.

This is my markup:

```html
<!DOCTYPE html>
<html>

<head>
  <title>Okta Sign-in Widget</title>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <!-- Latest CDN production CSS: 2.6.0 -->
    <link
        href="https://ok1static.oktacdn.com/assets/js/sdk/okta-signin-widget/2.6.0/css/okta-sign-in.min.css"
        type="text/css"
        rel="stylesheet"/>

    <!-- Theme file: Customize or replace this file if you want to override our default styles -->
    <link
        href="https://ok1static.oktacdn.com/assets/js/sdk/okta-signin-widget/2.6.0/css/okta-theme.css"
        type="text/css"
        rel="stylesheet"/>
</head>

<body>
  <div id="okta-login-container"></div>
  
  <!-- Latest CDN production JS: 2.6.0 -->
  <script
        src="https://ok1static.oktacdn.com/assets/js/sdk/okta-signin-widget/2.6.0/js/okta-sign-in.min.js"
        type="text/javascript"></script>

  <script type="text/javascript">
    var options = {
        baseUrl: 'http://rain.okta1.com',
        logo: 'http://rain.okta1.com:1802/assets/img/logos/okta-logo.png',
        features: {'securityImage': true},
        i18n: {
            'en': {
                'oform.errorbanner.title': 'TEST: We found some errors. Please review the form and make corrections.',
            },
        },
    };
    var signIn = new OktaSignIn(options);

    signIn.renderEl(
      { el: '#okta-login-container' },

      function success(res) {
        alert('Success');
      },

      function error(err) {
        alert('ERROR: ' + err);
      }
    );
  </script>
</body>

</html>
```

Screenshot where it works fine:

<img width="435" alt="screen shot 2018-02-15 at 4 58 34 pm" src="https://user-images.githubusercontent.com/24683447/36288881-d6360576-1271-11e8-840a-c98efe9483b5.png">

Can you tell me:

1. Browser/OS combination?
2. Browser size when this happens.
3. Your screen resolution.
4. Can we get a repo or a bare minimum markup where you can reproduce this?

Those would be really helpful in solving this.

**Comment by inergy** [comment]
@hor-kanchan-okta Yes, it was in fact a version issue. We were previously using 2.3.0 from our forked repo. This was an oversight on my part (whoops, sorry!)... thanks again!

@santhoshbalakrishnan The i18n issue was also resolved after updating to 2.6.0... sorry for the confusion!

**Comment by jmelberg-okta** [comment]
Hey @therealnicksaunders! Sounds like this issue can be resolved by updating to at least `2.6.0` of the widget. Appears to be related to the issue mentioned [here](https://github.com/okta/okta-signin-widget/issues/390#issuecomment-366111010).

If this issue persists, please provide us with a bit more information starting with:
 - Version of the widget you're using
 - Browser/OS

Closing this for now. Please reopen with the information mentioned above if you're still seeing this issue.
